### ISSUE TITLE
Re-order virtual desktops on Windows 10

### ISSUE THREAD

**Issue by JohnTravolski** [hwtnb/SylphyHornPlusWin11#13]
An older, no longer maintained [fork](https://github.com/ViRb3/SylphyHornEx) has a very useful keyboard shortcut that allows you to re-order Virtual Desktops for those of us still using Windows 10.

> Reorder virtual desktops
> (default key combination: Shift + Ctrl + Win + ← or →)
> 

From what I can tell, this shortcut / capability not implemented in this version for Windows 10.

Would it be possible to add this feature?

**Comment by sfranky** [comment]
i've never really used it, but there's a "swap desktop" feature in the settings, I would expect that's what it does, no ?

**Comment by mmikeww** [comment]
bump

**Comment by eggbean** [comment]
Yes, @sfranky is right. the "swap desktop" shortcuts do this.

I have the system tray icon set to show the desktop number and I have no OSD notification. I notice that the system tray icon number doesn't update after swapping desktop positions until changing desktop, but the desktops still swap.

(I use `PgUp` and `PgDn` instead of `Left` and `Right` for changing, swapping and moving windows in SylphyHornPlus11, so it matches changing/moving tabs on browsers and my tmux windows configuration. PgDn comes up as Next in this program, which seems to be a bug.)

![image](https://github.com/hwtnb/SylphyHornPlusWin11/assets/815311/6dabff93-bf8d-4ef1-b1f2-36df8add802d)

**Comment by mmikeww** [comment]
@eggbean yes obviously, but did both you and @sfranky even read the question, or the Readme:

> Creating and removing a desktop, and reordering (Windows 11 only) is available

I just tested on Win10 and those shortcuts are not able to be set. Unless I'm retarded. Can you please share how to do it on Win10?

**Comment by eggbean** [comment]
Oh, I missed where it says that it only works on Windows 11 and forgot that the question a was about Windows 10. The dark grey on slightly darker grey text doesn't help.

**Comment by mmikeww** [comment]
No worries. What is unusual is that the app linked in the first post was for win10 only, and claimed to have problems with win11. And the feature in this app is for win11 only and doesnt work for win10. I wonder why guys are having difficulty supporting both. Maybe an win api change or something
